src/android/CHIPTest/app/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
```

Comment:
Should we run CHIPTest in the Android CI? Ok to do as followup.

Reply:
Yes, in fact, I am already working on that locally.